Kickstarter-iOS/Views/Controllers/CancelPledgeViewController.swift

```diff
@@ -161,12 +194,28 @@ final class CancelPledgeViewController: UIViewController {
   // MARK: - Accessors
 
   @objc private func dismissKeyboard() {
-    self.view.endEditing(true)
+    self.viewModel.inputs.textFieldShouldReturn()
```

Comment:
Do we want to represent this event separately from `textFieldShouldReturn`?

Reply:
Ok you got me there.

Reply:
lol that's not the goal of my PR reviews 😂